Week4/Blockbuster/app/src/main/java/com/example/blockbuster/recyclerview/MovieViewHolder.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.blockbuster.recyclerview
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.blockbuster.R
+
+class MovieViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+    var poster: ImageView = itemView.findViewById(R.id.iv_item_poster)
```

Comment:
Try to avoid using findViewById, those have been replaced by view binding or synthetic properties.